helm/fdb-operator/values.yaml

```diff
@@ -1,12 +1,71 @@
-operator:
-  name: fdb-kubernetes-operator-controller-manager
-  image: foundationdb/fdb-kubernetes-operator
+---
+image:
+  repository: foundationdb/fdb-kubernetes-operator
   tag: v0.36.0
-  role: fdb-kubernetes-operator-manager-role
-  rolebinding: fdb-kubernetes-operator-manager-rolebinding
-  replicas: 1
-  globalMode:
-    enabled: false
+  pullPolicy: IfNotPresent
+
+initContainers:
+  resources:
+    limits:
+      cpu: 10m
+      memory: 50Mi
+    requests:
+      cpu: 10m
+      memory: 50Mi
+
+  init_6_1:
```

Comment:
Can we move this into a list we iterate over? Otherwise we have to change the helm chart again for `7.0.0` and all other new not minor releases.